File: iosgl/iosgl/bt/frame3d/dis/FrameBaseDisplay.m
//
//  FrameBaseDisplay.m
//  iosgl
//
//  Created by pan3dme on 2020/11/16.
//  Copyright © 2020 zhao. All rights reserved.
//

#import "FrameBaseDisplay.h"

@implementation FrameBaseDisplay
- (instancetype)scene3d:(Scene3D *)val
{
    self.scene3d=val;
    return self;
}
-(void)setFrameNodeUrl:(FrameNodeVo*)nodeVo;
{
    FrameBaseDisplay* this=self;
    this.frameNodeVo=nodeVo;
    this.groupItem =[[NSMutableArray alloc]init];
    Display3DSprite* dis = [[Display3DSprite alloc]init];
    [dis setObjUrl:nodeVo.resurl];
    [this.groupItem  addObject:dis];
}
@end

File: testproject/testdemo/app/src/main/java/com/pan3d/frame3d/dis/FrameBaseDisplay.java
package com.pan3d.frame3d.dis;

import com.pan3d.display.Display3DSprite;
import com.pan3d.frame3d.FrameNodeVo;
import com.pan3d.scene.Scene3D;

import java.util.List;

public class FrameBaseDisplay extends Display3DSprite {
    public boolean sceneVisible=false;
    protected FrameNodeVo frameNodeVo;
    public List<Display3DSprite> groupItem;
    public FrameBaseDisplay(Scene3D val) {
        super(val);
    }
    public void setFrameNodeUrl(FrameNodeVo $vo ) {
        this.frameNodeVo=$vo;

    }

    @Override
    public void upData() {
        super.upData();
    }
}